library/Icingadb/ProvidedHook/RedisHealth.php

```diff
@@ -39,10 +41,15 @@ public function checkHealth()
                 return;
             }
 
+            $this->setMetrics($server);
+
             $outdatedDbHeartbeat = $instance->heartbeat->getTimestamp() < time() - 60;
             if (! $outdatedDbHeartbeat || $instance->heartbeat->getTimestamp() <= $lastIcingaHeartbeat) {
                 $this->setState(self::STATE_OK);
-                $this->setMessage(t('Redis available and up to date.'));
+                $this->setMessage(sprintf(
+                    t('Redis available and up to date. (Version: %s)'),
+                    $server['redis_version'] ?? t('N/A')
```

Comment:
Since this message now contains a specific version, I would interpret this as the Redis server version is up to date. However, the guarding `if` checks the heartbeat age, not the package version. Thus, this can be misleading.

```suggestion
                    t('Data from Redis is up to date. (Version: %s)'),
                    $server['redis_version'] ?? t('N/A')
```